beetsplug/discogs.py

```diff
@@ -302,6 +303,13 @@ def get_album_info(self, result):
         mediums = [t.medium for t in tracks]
         country = result.data.get('country')
         data_url = result.data.get('uri')
+        style = result.data.get('styles')
+        if style is None:
+            self._log.info('Style not Found')
```

Comment:
I think this should probably be a debug message (it's probably not urgent enough to interrupt the `import` output).